lr_face/data.py

```diff
@@ -299,7 +299,65 @@ def images(self) -> List[FaceImage]:
         for file in files:
             path = os.path.join(self.RESOURCE_FOLDER, file)
             identity = f'FORENFACE-{file[:3]}'
-            data.append(FaceImage(path, identity, source=str(self)))
+
+            # parse annotations in filename:
+            # cx -> camera number
+            # a = no hat, b = hat
+            annotation_code = os.path.splitext(file)[0][3:]
+            if annotation_code in ('lp', 'rp'):
+                yaw = Yaw.PROFILE
+                pitch = Pitch.FRONTAL
+                headgear = False
+                low_quality = False
```

Comment:
is het een idee om hier ook de resolutie bins LOW, MEDIUM en GOOD aan te houden die ook in het PR van de calibratie strategieen zit?

Reply:
We kunnen als we het willen eventueel ook de uitgerekende quality gebruiken inplaats van deze annotatie. Dus ik laat hem er voor nu gewoon zo instaan.